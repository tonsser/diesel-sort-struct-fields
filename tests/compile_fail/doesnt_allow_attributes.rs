use diesel_sort_struct_fields::sort_fields;

#[sort_fields(foo)]
pub struct A {
    a: i32,
    b: i32,
}

fn main() {}
