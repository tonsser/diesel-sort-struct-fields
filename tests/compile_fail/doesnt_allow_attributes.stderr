error: `#[sort_fields]` doesn't support any attributes
 --> $DIR/doesnt_allow_attributes.rs:3:15
  |
3 | #[sort_fields(foo)]
  |               ^^^
